external/cblas/cblas.h

```diff
@@ -0,0 +1,577 @@
+/* CBlas header, downloaded from http://www.netlib.org/blas/#_cblas */
+
```

Comment:
You may not need cblas.h . I think all the blas libraries handles c++ demangling. So you may not need the extern C. you should be to directly call sgemv_ or dgemv_ , see the below example

Reply:
I don't follow what you are saying here, sorry!

Reply:
Done!